goshopify.go

```diff
@@ -285,6 +293,15 @@ func (c *Client) Do(req *http.Request, v interface{}) error {
 		}
 	}
 
+	if rl := resp.Header.Get("X-Shopify-Shop-Api-Call-Limit"); rl != "" {
+		if s := strings.Split(rl, "/"); len(s) == 2 {
+			c.RateLimits.RequestCount, _ = strconv.Atoi(s[0])
+			c.RateLimits.BucketSize, _ = strconv.Atoi(s[1])
```

Comment:
I'd prefer that we don't use blank identifiers for errors, we should probably handle this error.

Reply:
After discussing with @oliver006 I see his point there's not really a good way to handle this error, returning the default `0` is probably the safest bet so there is no need to check this error.